include/capstone/x86.h

```diff
@@ -300,23 +308,6 @@ typedef struct cs_x86 {
 	// Address size, which can be overrided with above prefix[5].
 	uint8_t addr_size;
 
-	// ModR/M byte
-	uint8_t modrm;
```

Comment:
yes, backward compatible is important, as we do not want to force so many apps to be rewritten.

another reason is that i think the encoding struct is solely for encoding info (all those offsets are to help you to find out where to patch the original input), so it makes more sense to me to keep sib/disp/etc in the original place.

thanks.